docroot/themes/humsci/humsci_basic/templates/layouts/three_column_w_image.html.twig

```diff
@@ -8,6 +8,14 @@
   {% set attributes = attributes.addClass(['hb-three-column-w-image--no-sidebar']) %}
 {% endif %}
 
+{% if content.left_sidebar and content.right_sidebar %}
+  {% set attributes = attributes.addClass(['hb-obj-three-column-w-image--two-sidebar']) %}
+{% elseif content.left_sidebar or content.right_sidebar %}
+  {% set attributes = attributes.addClass(['hb-obj-three-column-w-image--one-sidebar']) %}
+{% else %}
+  {% set attributes = attributes.addClass(['hb-obj-three-column-w-image--no-sidebar']) %}
+{% endif %}
+
```

Comment:
Sorry I didn't catch this earlier! It looks like lines 11 through 18 are duplicated.